src/BaGet/Startup.cs

```diff
@@ -113,10 +106,14 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 spa.Options.SourcePath = "../BaGet.UI";
 
-                if (env.IsDevelopment())
-                {
-                    spa.UseReactDevelopmentServer(npmScript: "start");
-                }
+                //if (env.IsDevelopment())
+                //{
+                //    spa.UseReactDevelopmentServer(npmScript: "start");
+                //}
```

Comment:
This React development server is really a headache for me. If it is enabled in all development environments, integration tests for my SPA file provider with custom path base fail. In addition to that we have a second middleware for serving SPA static files which should be disabled when using the React server.

My idea for a workaround would be to add a setting  `BaGetOptions.DisableReactDevelopmentServer`. Per default, all development environments would use the React server and unit tests would disabled it with an in-memory configuration. Any other ideas?